controllers/application_controller_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("Application Reconciler", func() {
 	var labelSet1 = map[string]string{"foo": "bar"}
 	var labelSet2 = map[string]string{"baz": "qux"}
 	var namespace1 = metav1.NamespaceDefault
-	var namespace2 = "default2"
+	var namespace2 = "kube-system"
```

Comment:
Not sure if we need this change?

Reply:
@tamalsaha Is this change necessary? https://github.com/kubernetes-sigs/application/pull/199/files#diff-973fe4443d3d0e4ae104b6deb46efed1R47

Reply:
Not if the tests pass. I could not find where this "default2" namespace was created.

Reply:
I’ll change it back to `default2`. Actually the value doesn't matter as it is only unit test. We won't check or create any k8s resources like namespaces in this test file.

Reply:
With controller-runtime upgraded from `v0.4.0` to `v0.6.0`, the test failed with `namespace not found` error. I explicitly created the namespace in the test to avoid using the `kube-system` namespace.